org.jacoco.tests/pom.xml

```diff
@@ -33,6 +33,7 @@
     <module>../org.jacoco.ant.test</module>
     <module>../jacoco-maven-plugin.test</module>
     <module>../org.jacoco.examples.test</module>
+    <module>../org.jacoco.tests.coverage</module>
```

Comment:
@manandbytes In fact I think that we should merge both "doc" and "coverage" into this module for distribution since they all share the same purpose and same nature. We can deal with this separately, however I also have feeling that your statement about classifiers reveals something in implementation of this new `report-aggregate`.